sources/engine/Xenko.Games/GameWindow.cs

```diff
@@ -228,6 +233,12 @@ protected void OnFullscreenToggle(object source, EventArgs e)
             handler?.Invoke(this, e);
         }
 
+        protected void OnClosing(object source, EventArgs e)
+        {
+            var handler = Closing;
+            handler?.Invoke(this, e);
```

Comment:
Should be inlined. `?` operator already does a `dup` in IL code.

Let's do it everywhere in that file.

Reply:
yes, i thought about this... but then decided against it, since there are many other places where this could be changed and i thought it's not the scope of this PR. but a general event code cleanup should be easy and quick to do with some code analyzer...